hub/config/src/main/java/uk/gov/ida/hub/config/resources/CountriesResource.java

```diff
@@ -10,23 +11,35 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import java.time.LocalDateTime;
 import java.util.Collection;
 
+import static uk.gov.ida.hub.config.Urls.ConfigUrls.EIDAS_DISABLED_AFTER_ROOT;
+
 @Path(Urls.ConfigUrls.COUNTRIES_ROOT)
 @Produces(MediaType.APPLICATION_JSON)
 public class CountriesResource {
 
     private final LocalConfigRepository<CountryConfig> countryConfigRepository;
+    private final ConfigConfiguration configuration;
 
     @Inject
     public CountriesResource(
-            LocalConfigRepository<CountryConfig> countryConfigRepository) {
+            LocalConfigRepository<CountryConfig> countryConfigRepository, ConfigConfiguration configuration) {
```

Comment:
I'd prefer to pass in the LocalDateTime on the constructor rather than the whole ConfigConfiguration. Be interested in @CharlesIC 's thoughts